dev/com.ibm.ws.cdi.interfaces/src/io/openliberty/cdi/spi/WebSphereCDIExtensionMetaData.java

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) 2015 IBM Corporation and others.
```

Comment:
copyright 2015->2020
Can you rename the file to be CDIExtensionMetadata? We should stop mentioning the WebSphere name.